sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/CHANGELOG.md

```diff
@@ -1,12 +1,22 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0.4 (Unreleased)
```

Comment:
will change to 1.0.4

Reply:
That's not likely to fly, either.   If this is not a GA release, it will require following the `-beta.#` convention.   If this is a GA, the archboard will need to approve and is likely to want justification for why we're not starting with 1.0.0, since this is the first GA.

Reply:
This change is getting ourselves ready for GA. Figured the unrelease next to the version would be the indication that this is not out for use yet. But if the process is to change the version last before GA, then i will remove this change and wait for the arch review.

Reply:
We'll want to wait until just before release and then change the version here and in the change log as well as add a change log date.   Since it's your first GA, you'd be 1.0.0 unless you come to another agreement with the architects during review.  Repository tooling uses the version to drive some automation rules and validation checks, so there is an impact to changing early.

Reply:
reverted